### PR TITLE
Written docs 3.0 and 5.1

### LINKED ISSUE #13: Create documentation 5,5.1,11,3

### PR DESCRIPTION
**Pull request by InigoGutierrez**
Closes #13 once pull request #18 is merged in.

### CODE REVIEW

**Comment by themrcesi** [comment]
First check done, tomorrow I'll do another one.

Looks nice!

File: src/docs/03_system_scope_and_context.adoc [hunk 1]
```diff
@@ -59,8 +67,4 @@ together with a mapping table showing the relationships between channels and inp
 
 ****
 
-**<Diagram or Table>**
-
-**<optionally: Explanation of technical interfaces>**
-
-**<Mapping Input/Output to Channels>**
+WIP (add only if it makes sense).
```
Comment (alvarezGarciaMarcos) [inline src/docs/03_system_scope_and_context.adoc:70]: What does WIP stand for?

Comment (alvarezGarciaMarcos) [reply src/docs/03_system_scope_and_context.adoc:70]: Okay, Work In Progress :sweat_smile:

File: src/docs/05_building_block_view.adoc [hunk 2]
```diff
@@ -57,17 +57,52 @@ In the best case you will get away with examples or simple signatures.
...
+}
+
+:User: --> react
+react --> map
+map --> routes
+map --> imaps
+routes --> solidmod
+react --> solidmod
+solidmod --> pod
+----
```
Comment (themrcesi) [inline src/docs/05_building_block_view.adoc]: It crashes when I test it locally!
crash

**Comment by InigoGutierrez** [comment]
The Whitebox Diagram can be improved, so this branch should not yet be merged.

File: src/docs/05_building_block_view.adoc (hunk 2 above)
Comment (themrcesi) [reply src/docs/05_building_block_view.adoc]: @garciaAlvarezPelayo @alvarezGarciaMarcos @alvarogarinf if you have the same problem, install: https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi

Comment (InigoGutierrez) [reply src/docs/05_building_block_view.adoc]: The solution to this problem was found: the dependency Graphviz was missing locally.

**Review by alvarezGarciaMarcos — COMMENTED** [review]
LGTM

**Pull request merged**